firestore/integration_test_internal/src/bundle_test.cc

```diff
@@ -199,6 +201,26 @@ TEST_F(BundleTest, LoadBundlesForASecondTimeSkips) {
   VerifyQueryResults(db);
 }
 
+TEST_F(BundleTest, LoadInvalidBundlesShouldFail) {
+  Firestore* db = TestFirestore();
+  std::vector<std::string> invalid_bundles{
+      "invalid bundle obviously", "\"(╯°□°）╯︵ ┻━┻\"",
+      "\0\1\2\3\ud7ff\ue000\uffff\""  // random bytes
```

Comment:
This new `LoadInvalidBundlesShouldFail` test does not appear to include an invalid utf-8 byte sequence (I tested out the sequences and they were all able to be decoded by Python's utf8 decoder). If this is true, please add an invalid UTF-8 byte sequence to these test scenarios, such as "\xc3\x28" (https://stackoverflow.com/questions/1301402/example-invalid-utf8-string).